mkdocs.yml

```diff
@@ -21,6 +21,7 @@ nav:
   - Guides:
       - Introduction: guides.md
       - Getting started: getting-started.md
+      - Simple Gateway: simple-gateway.md
```

Comment:
In addition to adding a reference here, it would be great to add this to the Guides page: https://github.com/kubernetes-sigs/service-apis/blob/master/docs-src/guides.md